safe/gui/tools/wizard/wizard_strings.py

```diff
@@ -74,38 +74,31 @@
     'raster to the hazard layer resolution during analyses. Please select the '
```

Comment:
I think this file would be better in definitions. But you don't have to change it.